packages/runtime/test/src/logicsig.ts

```diff
@@ -1,27 +1,66 @@
 import { decodeAddress, multisigAddress, MultisigMetadata } from "algosdk";
 import { assert } from "chai";
-
+const { convert } = require("@algo-builder/algob");
 import { AccountStore } from "../../src/account";
 import { RUNTIME_ERRORS } from "../../src/errors/errors-list";
 import { Runtime } from "../../src/runtime";
 import { useFixture } from "../helpers/integration";
 import { expectRuntimeError } from "../helpers/runtime-errors";
+const { types } = require("@algo-builder/web");
 
 const programName = "escrow.teal";
 const multiSigProg = "sample-asc.teal";
+const crowdFundEscrow = "crowdFundEscrow.teal";
 
 describe("Logic Signature", () => {
 	useFixture("escrow-account");
 	let john: AccountStore;
 	let bob: AccountStore;
 	let runtime: Runtime;
 	let johnPk: Uint8Array;
+	let goal: number;
+	let crowdfundApprovalFileName: string;
+	let crowdFundClearFileName: string;
+	let now: Date;
+	let beginDate: Date;
+	let endDate: Date;
+	let fundCloseDate: Date;
```

Comment:
you should declare variables like `goal`, `now`, `beginDate`, `endDate`, `fundCloseDate` in a smaller describe

Reply:
why so??? I can see in other tests we have used the same code format case.

Reply:
Well those variable are not use globally but only use within `crowdfundingApp`
You should put those here only if you use globally

Reply:
Done

Reply:
We need them global bcz it will be initialized before running each unit test. This is standard practice while writing unit test case.